app/controllers/webhooks_controller.rb

```diff
@@ -36,4 +36,27 @@ def gocardless
 
     head(:ok)
   end
+
+  def adyen
+    signature = adyen_params.dig("additionalData", "hmacSignature")
+
+    result = PaymentProviders::AdyenService.new.handle_incoming_webhook(
+      organization_id: params[:organization_id],
+      body: adyen_params
+    )
+
+    unless result.success?
+      if result.error.code == 'webhook_error'
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://rubystyle.guide#if-as-a-modifier)